docs/admin/runai-setup/cluster-setup/cluster-prerequisites.md

```diff
@@ -270,22 +270,20 @@ kubectl patch configmap/config-features \
 
 #### Inference Autoscaling
 
-Run:ai allows to autoscale a deployment according to various metrics:
+Run:ai allows to autoscale a deployment using the following metrics:
```

Comment:
Now that we're only left with these 2 metrics that are built-in and don't require any 3rd party installations, I don't see a reason to even mention the metrics in this context. It should only be described on the inference workload page. (But keep it commented-out as we'll soon need it back)